package.json

```diff
@@ -53,6 +53,8 @@
     "@angular/router": "3.0.0",
     "@angular2-material/core": "^2.0.0-alpha.8-2",
     "core-js": "^2.4.1",
+    "handlebars": "^4.0.5",
+    "handlebars-dateformat": "^1.0.1",
```

Comment:
we do not use npm now, but probably you can include dateformat from cdn and use it